.gitignore

```diff
@@ -45,6 +45,8 @@ swupd_*
 swupd.bash
 tap-driver.sh
 test-driver
+test-bundle*
+test-file*
```

Comment:
Some of the tests use files with these names, and they are committed to the repo. So, you have the option of making these patterns more specific to certain paths, or you can add a .gitignore instead the test directories that need it.